packages/core/demo/chart-types.ts

```diff
@@ -15,7 +15,9 @@ const {
 	curvedLineData,
 	lineData,
 	lineOptions,
+	// Scatter
```

Comment:
Unused?
```suggestion
```

Reply:
Just a comment, ever other chart-type has a comment as well. Line and scatter are two different implementations.